carpics_360.js

```diff
@@ -0,0 +1,540 @@
+CarPicsSpinnerAPI = (function() {
+    this.CarPicsSpinners = function(config) {
+        this.spinners = {};
+        this.makeBoundGetRequest = function(config, thisObj) {
+            return function(data) {
+                if (data.length==0){
+                    return;
+                }
+                thisObj.spinners[config.divId].spinner = new CarPicsSpinner(config, data);
+                thisObj.spinners[config.divId].spinStatus = false;
+            }
+        }
+        this.AddSpinner = function(config, urlArray) {
+            this.spinners[config.divId] = {
+                spinner: new CarPicsSpinner(config, urlArray),
+                spinStatus: false
+            };
+        }
+        this.MakeSpinner = function(config) {
+            this.spinners[config.divId] = {
+                spinStatus: false,
+                spinner: {}
+            }
+            $.get(config.sourceURL, {}, this.makeBoundGetRequest(config, this), "json");
+        }
+        for (var i = 0; i < config.spinners.length; i++) {
+            this.MakeSpinner(config.spinners[i]);
+        }
+    }
+
+    this.CarPicsSpinner = function(config, data) {
```

Comment:
Not a function name.  This is idiomatic to browser JavaScript.  This is declaration of a class.